.packit.yaml

```diff
@@ -0,0 +1,12 @@
+actions:
+  get-current-version:
+    - "python3 ./setup.py --version"
```

Comment:
You should be able to replace this by https://packit.dev/docs/configuration/#upstream_tag_template.

```
upstream_tag_template: blivet-{version}-1
```

Reply:
That's probably not going to work for us with the correct version only on the `-release` branch.

Reply:
OK ;)